backend/src/endpoints/user.py

```diff
@@ -6,8 +6,7 @@
 from pydantic.main import BaseModel
 from sqlalchemy.orm import Session
 from src.database import session_for_request
-from src.database.user import get_user_by_id
-from src.database.user import get_user_missions_by_wallet
+from src.database.user import get_user_missions_by_wallet, get_leaderboard, get_user_by_id
```

Comment:
```suggestion
from src.database.user import get_leaderboard
from src.database.user import get_user_by_id
from src.database.user import get_user_missions_by_wallet
```

Each import should be on a different line, that way makes the pull requests cleaner when adding or removing dependencies, and also avoids merge conflict when 2 o more are working on the same file.